src/frontend/servers/y-provider/package.json

```
@@ -39,13 +39,14 @@
     "@types/ws": "8.18.1",
     "cross-env": "7.0.3",
     "eslint-config-impress": "*",
-    "jest": "30.0.3",
     "nodemon": "3.1.10",
     "supertest": "7.1.1",
     "ts-jest": "29.4.0",
     "ts-node": "10.9.2",
     "tsc-alias": "1.8.16",
     "typescript": "*",
+    "vitest": "^3.2.4",
+    "vitest-mock-extended": "^3.1.0",
```

We pin the dependency, we have "renovate" managing the bumping.

```suggestion
    "vitest": "3.2.4",
    "vitest-mock-extended": "3.1.0",
```

Can you remove `ts-jest` and `@types/jest` dependency as well ?

---

> Can you remove ts-jest and @types/jest dependency as well ?

I did and forcepushed.

---

What about `ts-jest` ?